modules/mo_pytorch/README.md

```diff
@@ -0,0 +1,38 @@
+# PyTorch extension for Model Optimizer
+
+This module let you convert PyTorch models directly to OpenVINO IR without using ONNX
+
+## Usage
+
+1. Clone repository
+
+    ```bash
+    git clone --depth 1 https://github.com/openvinotoolkit/openvino_contrib
+    ```
+
+2. Setup environment
+
+    ```bash
+    source /opt/intel/openvino_2021/bin/setupvars.sh
```

Comment:
may be we can replace `2021` with abstract placeholder for year?

Reply:
I'm confused why it's not `/opt/intel/openvino` for 2021.1 release

Reply:
need to ask @moslex @artanokhov

Reply:
Replaced to more generic `source /opt/intel/openvino_<VERSION>/bin/setupvars.sh` which is true for old and new releases.